### PR TITLE
Cypress CI Fix

### PR DESCRIPTION
**Pull request by ug02fast**
This PR integrates the `currents` hosted test-runner to allow developers to have faster feedback loops on test failures. Having videos saved will save a great amount of time from having to context switch prematurely and identify false-positives.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -87,7 +87,10 @@ jobs:
         image: saddlefinance/contracts:latest
         ports:
           - 8545:8545
-
+      saddle-token:
```
Comment (hammeiam) [inline .github/workflows/ci.yml:90]: Rm

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -107,6 +110,22 @@ jobs:
...
+      # Save videos and screenshots as test artifacts
+      # https://github.com/actions/upload-artifact
+      - uses: actions/upload-artifact@master
+        with:
+          name: screenshots
+          path: cypress/screenshots
+          # there might be no screenshots created when:
+          # - there are no test failures
+          # so only upload screenshots if previous step has failed
+          if: failure()
```
Comment (hammeiam) [inline .github/workflows/ci.yml:123]: Awesome!

File: cypress.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
+  "projectId": "ch3eek",
```
Comment (hammeiam) [inline cypress.json:2]: Will we have access to the project on Cypress since they're hosting the videos? What I'd you're out of town and run out of space

**Review by hammeiam — APPROVED** [review]

File: cypress.json (hunk 3 above)
Comment (ug02fast) [reply cypress.json:2]: Going to send you invite link ser

**Comment by ug02fast** [comment]
closed in favor of #884

**Pull request closed without merging**